### DISCUSSION TITLE
Labels

### DISCUSSION THREAD

**Discussion by feersantana5** [isl-org/Open3D#3728] [Q&A] [answered]
How is possible to add text in the point cloud like object labels?

**Comment by germanros1987** [comment, marked as answer]
HI @feersantana5 through open3d.visualization.O3DVisualizer and also through open3d.visualization.gui.SceneWidget you can call the method add_3d_label. Further information can be found here: http://www.open3d.org/docs/latest/python_api/open3d.visualization.gui.SceneWidget.html?highlight=add_3d_label
